cachi2/interface/cli.py

```diff
@@ -59,6 +60,14 @@ def version_callback(value: bool) -> None:
     raise typer.Exit()
 
 
+def _yaml_validate(file: str) -> None:
```

Comment:
The type of this parameter here is actually `pathlib.PosixPath`

Reply:
it's `pathlib.Path` I believe (which happens to be PosixPath on Linux)

Side note, the function does not validate yaml :smile: